index.js

```diff
@@ -180,13 +180,23 @@ module.exports = function(version) {
                 instruction = instructionObject.default;
             }
 
+            var destinations = step.destinations && step.destinations.split(': ');
```

Comment:
I assume we can always split on `': '` because this value is generated from OSRM, yes?

Reply:
Yes, `': '` is inserted by OSRM:

https://github.com/Project-OSRM/osrm-backend/blob/fd57c5b48bd289ecf15ffa7ea703f0461536f784/profiles/lib/destination.lua#L23

For the other calls to `split()`, I passed in `','` instead of `', '`, because `tokenize()` already trims whitespace from each token:

https://github.com/Project-OSRM/osrm-text-instructions/blob/ad11173324c91e30a014ab61c23bb5043d130236/index.js#L215

A colon is also used [to mark a possessive or contraction](https://en.wikipedia.org/wiki/Colon_%28punctuation%29#Word-medial_separator) in Finnish and Swedish, so I included a space in `': '` just in case.

Reply:
👍